draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -0,0 +1,1037 @@
+    
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-00</t>
+            <t><list style="symbols">
+    		<t>We added a feature to allow more than one post-quantum key
+            exchange algorithms to be negotiated and used to exchange a post-
+            quantum shared secret.</t>
+    		<t>Instead of relying on TCP encapsulation to deal with IP level
+    		fragmentation, we introduced a new key exchange payload that can
+    		be sent as multiple fragments within IKE_SA_INIT message.</t>
+    	    </list>
+            </t>
+        </section>
+    
+    	<section title="Document Organization" >
+            <t>
+            The remainder of this document is organized as follows.  <xref target="design"/>
+            summarizes design criteria.  <xref target="framework"/> describes how
+    	    multiple key exchanges are performed between two IKE peers and how
+    	    keying materials are derived for both SAs and child SAs.  A summary of alternative
+    	    approaches that have been considered, but later discarded, are described
+    	    in <xref target="altdesign"/>.  <xref target="IANA"/>
+            discusses IANA considerations for the namespaces introduced in this
+    	    document, and lastly <xref target="security"/> discusses security considerations.
+            </t>
+    
+            <t> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", 
+            "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted 
+            as described in BCP 14 <xref target="RFC2119" /> <xref target="RFC8174" /> when, and only when, 
+            they appear in all capitals, as shown here.
+            </t>
+     	</section>
+	</section>
+
+	<section title="Design Criteria" anchor="design">
+        <t>
+        The design of the proposed framework is driven by the
+        following criteria:</t>
+    
+    	<t><list style="hanging" hangIndent="5"><t hangText="1)">
+    	Need for post-quantum cryptography in IPsec.  Quantum computers
+    	might become feasible in the next 5-10 years.  If current
+        Internet communications are monitored and recorded today (D),
+        the communications could be decrypted as soon as a quantum-
+        computer is available (e.g., year Q) if key negotiation only
+        relies on non post-quantum primitives.  This is a high threat
+        for any information that must remain confidential for a long
+        period of time T &gt; Q-D.  The need is obvious if we assume that Q
+        is 2040, D is 2020, and T is 30 years.  Such a value of T is
+        typical in classified or healthcare data.
+    	</t>
+    
+    	<t hangText="2)">
+    	Hybrid.  Currently, there does not exist a post-quantum key
+    	exchange that is trusted at the level that ECDH is trusted
+        against conventional (non-quantum) adversaries.  A hybrid
+        approach allows introducing promising post-quantum candidates
```

Comment:
I think the use of the word candidates implies that we only need hybrid while NIST is standardising PQC. However, it is quite likely that hybrid may still be used even after standardisation. As we stated in some parts of this documents, some of the PQC algorithms are relative new and are not that well-trusted yet. So how about this change:

```suggestion
        post-quantum algorithms to be introduced
```